RPIPs/RPIP-49.md

```diff
@@ -15,38 +15,54 @@ This Info RPIP exists to explain an overall tokenomics proposal, the RPIPs it ha
+  - Another possibility discussed is using RPL inflation to fulfill the role currently done by "voter share"
+- Cashing out credit as rETH
+- Interim solutions (note: we can better consider these _after_ we've established a rough goal and a timeline)
+
+## Topics not being addressed in this rework
+- MEV penalty improvements - this _is_ important, but the numbers we've seen previously suggest MEV theft has not been a pressing problem.
+- rETH protection - this is moderate priority and significant complexity. While I wish to see it, I don't think it's worth risking timeline on the [components above](#components-currently-being-advocated-for).
+- Right now voter share is split up amongst potential voters based on amount of RPL eligible to vote. We could do something based on active governance or actual vote power instead of count of RPL?
+  - Possible thought: require having a delegate set for eligibility
+  - Possible thought: setting a delegate expires after a time period (eg, 1 year)
+  - Possible thought: if a delegate has x% of vote power, prevent new delegations to them (revert with error)
+  - Simplicity is being favored more than these; however, this may be something to revisit, depending on the actual practice of governance observed
 
 ## Potential deployment scenarios
-We have some thought experiment scenarios. For all of these, the premise is that we'd do the governance for the whole Scenario before starting.
+We have some thought experiment scenarios. For all of these, the premise is that we'd do the governance for the whole Scenario before starting. We'd love to get team feedback here, especially on ballpark timelines.
 
-We'd love to get team feedback here, especially on ballpark timelines.
+In order to avoid messy conditionals in the rest of the RPIPs, we will be operating with "Scenario C" as the best guess.
 
 ### Scenario A (1 release):
 - Release 1: Components 1-6 at once.
 
 ### Scenario B (3 releases):
-- Release 1: Megapools, Remove operator-gated upgrades
+- Release 1: Megapools, Forced delegaet upgrades
   - Allows ETH-only participants
-  - Allows NOs to migrate to 4-ETH minipools for greater efficiency
   - Allows NOs to move to Megapools for greater efficiency
-    - Note that this implies very high earnings for ETH-only minipools until Release 2
+  - Allow 4-ETH validators within Megapools
 - Release 2: RPL burn, UVC
   - New RPL value capture comes online
   - Revenue steering controls come online
 - Release 3: Bond curves, Forced exits
   - Dramatically improve capital efficiency 
 
-### Scenario C (2 releases):
-- Release 1: Megapools, Remove operator-gated upgrades, RPL burn, UVC
+### Scenario C (2 releases; currently assumed for all RPIPs):
+- Release 1: Megapools, Forced delegaet upgrades, RPL burn, UVC
   - Allows ETH-only participants
   - Allows NOs to migrate to 4-ETH minipools for greater efficiency
```

Comment:
Change language to match Scenario B:
- Allows NOs to move to Megapools for greater efficiency
- Allow 4-ETH validators within Megapools

Avoids confusion compared to "migrating to 4-ETH minipools"